controllers/error.controller.js

```diff
@@ -0,0 +1,3 @@
+exports.errorHandler = (err, req, res, next) => {
+    next(err);
```

Comment:
You are calling 'next' in this last middleware, but there's no more. 

Instead you could use app.all('*',()=>{}) to catch any bad path.

https://www.geeksforgeeks.org/express-js-app-all-function/